store-vault/src/test/groovy/com/mx/path/facility/store/vault/VaultStoreConfigurationTest.groovy

```diff
@@ -0,0 +1,31 @@
+package com.mx.path.facility.store.vault
+
+import java.time.Duration
+
+import spock.lang.Specification
+
+class VaultStoreConfigurationTest extends Specification {
+  def "GetRetryInterval"() {
+    given:
+    def subject = new VaultStoreConfiguration()
+
+    when:
+    subject.setRetryIntervalMilliseconds(531)
+
+    then: "uses milliseconds"
+    subject.getRetryInterval() == Duration.ofMillis(531)
```

Comment:
in groovy you can directly access the property instead of using a getter, ie `subject.retryInterval`

Reply:
True, but this test is literally for that getter. So, I think it's a little clearer to use it here. This is temporary anyways. 👍